### DISCUSSION TITLE
Playlist Track Items Always Null

### DISCUSSION THREAD

**Discussion by tsar-boomba** [JohnnyCrazy/SpotifyAPI-NET#739] [Q&A]
Every time I try to access the tracks of a playlist they are null. What am I doing wrong?
```
_spotify = _spotifyService.Spotify;
_me = await _spotify.UserProfile.Current();

// playlists where user is owner
_playlists = (await _spotify.PaginateAll(await _spotify.Playlists.CurrentUsers()))
    .Where(playlist => playlist.Owner.Id == _me.Id);
```

**Comment by JohnnyCrazy** [comment]
Did you request the correct scopes?

[playlist-read-private](https://developer.spotify.com/documentation/general/guides/authorization/scopes/#playlist-read-private)

**Reply by tsar-boomba** [reply]
These are the scopes my client is using
```Scopes.PlaylistReadPrivate, Scopes.PlaylistReadCollaborative```

**Comment by JohnnyCrazy** [comment]
Is the list empty or are the items of the playlists `null`?

**Reply by tsar-boomba** [reply]
The Tracks.Items field is ```null```.

**Reply by ta264** [reply]
Just spotted this.  See #779.  By default I think everything will be null unless you explicitly ask for it via `Fields`

**Reply by JohnnyCrazy** [reply]
Per Default, all fields should be fetched. If not, it's definitely a bug
